css/at-rules/media.json

```diff
@@ -611,17 +611,22 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/forced-colors",
             "spec_url": "https://drafts.csswg.org/mediaqueries-5/#forced-colors",
             "support": {
-              "chrome": {
-                "version_added": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#forced-colors",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "See <a href='https://crbug.com/970285'>bug 970285</a>."
-              },
+              "chrome": [
+                {
+                  "version_added": "79",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "#forced-colors",
+                      "value_to_set": "Enabled"
+                    }
+                  ],
+                  "notes": "See <a href='https://crbug.com/970285'>bug 970285</a>."
+                },
+                {
+                  "version_added": "89"
```

Comment:
Unfortunately the order of entries matter, and the first one is the one that'll be shown on MDN by default. So you'll need to change the order to put this first.

Reply:
Oops! Fixed. Thanks for bearing with me!